src/request/notification/default.rs

```diff
@@ -1,8 +1,25 @@
 use crate::request::notification::{NotificationBuilder, NotificationOptions};
-use crate::request::payload::{APSAlert, Payload, APS};
+use crate::request::payload::{APSAlert, APSSound, Payload, APS};
 
 use std::{borrow::Cow, collections::BTreeMap};
 
+fn is_zero(value: &u8) -> bool {
+    *value == 0
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+#[serde(rename_all = "kebab-case")]
+pub struct DefaultSound<'a> {
+    #[serde(skip_serializing_if = "is_zero")]
+    critical: u8,
```

Comment:
Can we abstract the fact that critical is a u8, or do we need to support it being any u8 int?

Reply:
Sure, we can abstract